node-backend/app/controllers/v1/loginController.js

```diff
@@ -64,12 +64,14 @@ module.exports.login = async (req, res) => {
       });
     } else if (domainResult) {
       let orgId = domainResult.id;
+      let hi5QuotaBalance = domainResult.hi5_limit;
       let checkerror = await insertData(
         orgId,
         firstName,
         lastName,
         email,
-        displayName
+        displayName,
+        hi5QuotaBalance
```

Comment:
domainResult.hi5_limit can be directly used here instead of getting into new variable. Same for display name as well